.github/workflows/tests.yml

```diff
@@ -14,8 +14,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  conda:
-    name: Build (and upload) - ${{ matrix.os }}, Python ${{ matrix.python-version }}, Runtime ${{ matrix.runtime-version }}
+  tests:
+    name: ${{ matrix.os }}, Python ${{ matrix.python-version }}, Runtime ${{ matrix.runtime-version }}
```

Comment:
This makes the title for PR checks nicer. Currently they look like

```
Conda build / Build (and upload) - ubuntu-latest, Python 3.7, Runtime latest
```

with the changes here they look like

```
Tests / ubuntu-latest, Python 3.7, Runtime latest
```